src/Workspaces/Core/Portable/CodeFixes/CodeFixContext.cs

```diff
@@ -48,7 +48,13 @@ public struct CodeFixContext
         /// </summary>
         /// <param name="document">Document to fix.</param>
         /// <param name="span">Text span within the <paramref name="document"/> to fix.</param>
-        /// <param name="diagnostics">Diagnostics to fix. All the diagnostics should have the same span <paramref name="span"/>.</param>
+        /// <param name="diagnostics">
+        /// Diagnostics to fix. All the diagnostics should have the same span <paramref name="span"/>.
+        /// <para>
+        /// All the provided diagnostics must have IDs that are fixable by the associated <see cref="CodeFixProvider"/>, as indicated by its <see cref="CodeFixProvider.FixableDiagnosticIds"/> property.
```

Comment:
How about: "The ID of each diagnostic must be in the set of the FixableDiagnosticIds of the associated CodeFixProvider" (with crefs included as appropriate)?